### PR TITLE
Egate 73: Update Hello World APP to use ActiveMQ

### PR DESCRIPTION
**Pull request by jamalkm**
Updated the app to use ActiveMQ as the broker.
The broker URL and the topic name are retrieved from the Event-Gateway Topic Endpoint REST API.

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -5,8 +5,18 @@ spring.jpa.hibernate.ddl-auto=create-drop
...
 
-messaging.kafka.enabled=false
-messaging.kafka.applicationId=Hello-World-Kafka-Streams
-messaging.kafka.bootstrapServers=localhost:9092
-# Comma separated list of topics
-messaging.kafka.sourceTopics=alfresco.repo.events.nodes
\ No newline at end of file
+messaging.config.enabled=false
+
+messaging.config.topic.baseUrl=http://localhost:8080
```
Comment (rgauss) [inline src/main/resources/application.properties]: Perhaps we should start to distinguish these properties from the many others that a developer would encounter when building a larger Spring Boot app and change to something like:

`alfresco.events.eventGateway.url`

then hardcode the topic endpoint as constant.  As mentioned above we'd like to move this class to an SDK and don't want to confuse developers with being able to tinker with endpoints.

The fallback properties could just be `alfresco.events.broker.*`, `alfresco.events.topic`.

File: src/main/java/org/alfresco/deployment/sample/activemq/CamelRouteConfig.java [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+import org.springframework.retry.policy.SimpleRetryPolicy;
+import org.springframework.retry.support.RetryTemplate;
+import org.springframework.web.client.RestTemplate;
+
+/**
+ * @author Jamal Kaabi-Mofrad
+ */
+@Configuration
+@ConditionalOnProperty(prefix = "messaging.config", name = "enabled", havingValue = "true")
+public class CamelRouteConfig
```
Comment (rgauss) [inline src/main/java/org/alfresco/deployment/sample/activemq/CamelRouteConfig.java]: This is the type of class we'd like to see in an Events SDK in the very near future.

Perhaps we should pull them into a package that would make it easier to move when we do that, or even go so far as to make it a separate Maven module or even separate repo now?  (that may be more work than we sized for this story though)

File: src/main/resources/application.properties (hunk 1 above)
Comment (jamalkm) [reply src/main/resources/application.properties]: Sure, sounds good!

File: src/main/java/org/alfresco/deployment/sample/activemq/CamelRouteConfig.java (hunk 2 above)
Comment (jamalkm) [reply src/main/java/org/alfresco/deployment/sample/activemq/CamelRouteConfig.java]: I guess it is outside of this sprint's scope now. I purposely encapsulated all the required logic in one class so we can move it easily.

File: src/main/resources/application.properties (hunk 1 above)
Comment (jamalkm) [reply src/main/resources/application.properties]: Done!

**Pull request merged**